### PR TITLE
[TEST][CI] Enable pr-code-format on branches in intel/llvm (#13101)

### PR DESCRIPTION
**Pull request by npmiller**
Testing to see if this trigger the CI, we may need to cherry-pick the branch matching on the release branch

### CODE REVIEW

**Comment by npmiller** [comment]
That didn't seem to trigger any extra check, @aelovikov-intel do you know if we need to have the config changes on `sycl-rel-5_2_0`, this PR didn't trigger anything either: https://github.com/intel/llvm/pull/13353

File: .github/workflows/pr-code-format.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ on:
   pull_request_target:
     branches:
       - main
+      - sycl
+      - sycl-devops-pr/**
```
Comment (aelovikov-intel) [inline .github/workflows/pr-code-format.yml:7]:
```suggestion
```

**Comment by aelovikov-intel** [comment]
> That didn't seem to trigger any extra check, @aelovikov-intel do you know if we need to have the config changes on `sycl-rel-5_2_0`, this PR didn't trigger anything either: #13353

This workflow has 'pull_request_target' trigger, so the change has to be merged first to take any effect.

**Comment by npmiller** [comment]
That makes sense, it is merged in the `sycl` branch, but didn't make it into the `sycl-rel-5_2_0` branch, that's why I was wondering if it might work anyway or if we need to cherry-pick.

I'll gather the patches that updated this and open a big cherry-pick PR

**Comment by npmiller** [comment]
Actually this was the only pre-commit workflow that didn't have the branch matching, the others have it but the jobs aren't getting triggered anyway, so not sure what's going on then

**Pull request closed without merging**